app/screens/move-money-screen/move-money-screen.tsx

```diff
@@ -225,16 +224,20 @@ export const MoveMoneyScreenDataInjected: ScreenType = ({
   })
 
   const mobileVersions = data?.mobileVersions ? data.mobileVersions[0] : undefined // FIXME array/item mismatch
-  const mergedTransactions = data?.me?.defaultAccount?.transactions?.edges
+  const transactionsEdges = data?.me?.defaultAccount?.transactions?.edges ?? undefined
   const usdWalletId = data?.me?.defaultAccount?.usdWallet?.id
 
   const btcWalletValueInUsd = hasToken
-    ? data?.me?.defaultAccount?.btcWallet?.usdBalance
+    ? data?.me?.defaultAccount?.btcWallet?.usdBalance ?? NaN
```

Comment:
I think this code can be simplified.
In a sense adding `: 0` does not change the final value of `btcWalletValueInUsd` when `hasToken` is false because it'll always fallback to `NaN` not 0. we could just write it as:
```js
const btcWalletValueInUsd = hasToken ? data?.me?.defaultAccount?.btcWallet?.usdBalance : NaN

The same applies for `usdWalletBalance` and `btcWalletBalance`

Reply:
if `hasToken` is `true` but the value is undefined (should not be, but is required for type) then the value will be `NaN`

Reply:
if `hasToken` is `false`, then the value will be `0`